packages/ember-views/lib/compat/attrs-proxy.js

```diff
@@ -56,46 +37,34 @@ let AttrsProxyMixin = {
     val.update(value);
   },
 
-  willWatchProperty(key) {
-    if (this._isAngleBracket || key === 'attrs') { return; }
-
-    let attrsKey = `attrs.${key}`;
-    _addBeforeObserver(this, attrsKey, null, attrsWillChange);
-    addObserver(this, attrsKey, null, attrsDidChange);
+  _propagateAttrsToThis() {
+    let attrs = this.attrs;
+    let values = {};
+    for (let prop in attrs) {
+      if (prop !== 'attrs') {
+        values[prop] = this.getAttr(prop);
+      }
+    }
+    this.setProperties(values);
   },
 
-  didUnwatchProperty(key) {
-    if (this._isAngleBracket || key === 'attrs') { return; }
-
-    let attrsKey = `attrs.${key}`;
-    _removeBeforeObserver(this, attrsKey, null, attrsWillChange);
-    removeObserver(this, attrsKey, null, attrsDidChange);
+  didReceiveAttrs() {
+    this._isDispatchingAttrs = true;
```

Comment:
super?



lets make sure _isDispatchingAttrs is in the original shape of the object (set during init)